diff --git a/config.def.h b/config.def.h
index 100093e..d306816 100644
--- a/config.def.h
+++ b/config.def.h
@@ -66,5 +66,8 @@ static const char unknown_str[] = "n/a";
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+    { battery_perc, "BAT %s | ",    "BAT0" },
+    { wifi_essid, "%s | ",          NULL},
+    { vol_perc, "VOL %s | ",        "default" },
+	{ datetime, "%s",               "%F %T" },
 };
